phenol-io/src/test/java/org/monarchinitiative/phenol/io/OntologyLoaderTest.java

```diff
@@ -21,6 +22,17 @@
  */
 class OntologyLoaderTest {
 
+  private static Ontology hpo;
+
+  @BeforeAll
```

Comment:
just make this a private final field of the class - no need to use any Junit gubbins for this.